homeassistant/components/uptime_kuma/binary_sensor.py

```diff
@@ -0,0 +1,43 @@
+"""Summary binary data from Uptime Kuma."""
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
+    """Defer sensor setup to the shared sensor module."""
+    coordinator = hass.data[DOMAIN]
+    async_add_entities(
+        UptimeKumaBinarySensor(coordinator, monitor) for monitor in coordinator.data
+    )
+
+
+class UptimeKumaBinarySensor(BinarySensorEntity, CoordinatorEntity):
+    """Represents an Uptime Kuma binary sensor."""
+
```

Comment:
Set the icon as a class constant
```suggestion

    _attr_icon = "mdi:cloud"

```